draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -630,6 +671,10 @@ The following Key Binding JWT payload was created and signed for this presentati
 
 <{{examples/simple/kb_jwt_payload.json}}
 
+If the Verifier did not require Key Binding, then the Holder could have
+presented the SD-JWT from (#issuance) directly, instead of encapsulating it in
+an SD-JWT-KB.
+
```

Comment:
the SD-JWT from (#issuance) includes many more Disclosures than the example in this section. So this sentence isn't quite right and could be confusing/misleading. Meanwhile I don't think what it's trying to say is needed right here. So:

```suggestion
```

Reply:
is there a place where we can move this paragraph to? instead of deleting it

Reply:
The suggestion to delete was b/c of it being not quite right. This tries to fix that. 
```suggestion
If the Verifier did not require Key Binding, then the Holder could have
presented the SD-JWT with selected Disclosures directly, instead of encapsulating it in
an SD-JWT-KB.

```